syntax = "proto3";

package burgerRpc;

option cc_generic_services = true;

message ResultCode {
    int32 errcode = 1;
    bytes errmsg = 2;
}

message LoginRequest {
    bytes name = 1;
    bytes pwd = 2;
}

message LoginResponse {
    ResultCode result = 1;
    bool success = 2;
}

// 最重要的就是 函数名 函数参数 返回 三个部分
service UserServiceRpc {
    // 不一定和函数名相等，但是最好对应起来
    rpc Login(LoginRequest) returns(LoginResponse);
}
